docker/Dockerfile

```diff
@@ -0,0 +1,23 @@
+# syntax=docker/dockerfile:1.4
+
+ARG BASE_IMAGE
+
+FROM ${BASE_IMAGE}
+
+ARG IMAGE_ENTRYPOINT="exec java -Dlog4j2.debug=false -Dlog4j.configurationFile=log4j2-console.xml -jar /opt/override/crypto.jar"
```

Comment:
Note we do not have `log4j2-console.xml` file in this project.

Reply:
dealt with that